documentation/modules/ROOT/pages/operations/debezium-operator.adoc

```diff
@@ -0,0 +1,831 @@
+
+This document provides a high-level overview of the operator's architecture, components, and capabilities. For detailed configuration options, see Configuration Reference. For practical deployment examples, see Examples and Use Cases. For installation procedures, see Installation and Deployment.
+
+== DebeziumServer custom resource
+The DebeziumServer custom resource provides a declarative way to configure Debezium Server instances.
+
+=== Basic Structure
+[source,yaml]
+apiVersion: debezium.io/v1alpha1
+kind: DebeziumServer
+metadata:
+  name: my-debezium-server
+spec:
+  image: String
+  version: String
+  runtime:
+    environment:
+      vars:
+        - name: String
+          value: String
+      from:  # EnvFromSource array
+        - sourceName: String
+          # add other EnvFromSource fields as needed
+    storage:
+      data:
+        type: persistent | ephemeral
+        claimName: String  # required if type is "persistent"
+      external:  # Volume array
+        - name: String
+          # add other Volume fields as needed
+    jmx:
+      enabled: boolean
+      port: int  # defaults to 1099
+    templates:
+      container:
+        resources: ResourceRequirements
+        securityContext: SecurityContext
+      pod:
+        metadata:
+          annotations:  # Map<String, String>
+            key: value
+          labels:  # Map<String, String>
+            key: value
+        imagePullSecrets:  # List
+          - name: String
+        affinity: Affinity
+        securityContext: PodSecurityContext
+  quarkus:
+    config:
+      # quarkus properties
+      format:
+        value:
+          type: String
+          config:
+            # format properties
+      key:
+        type: String
+        config:
+          # format properties
+      header:
+        type: String
+        config:
+          # format properties
+  transforms:
+    - type: String
+      predicate: String
+      negate: Boolean
+      config:
+        # transformation properties
+  predicates:
+    name:
+      type: String
+      config:
+        # predicate properties
+  sink:
+    type: String
+    config:
+      # sink properties
+  source:
+    class: String
+    config:
+      # source connector properties
+
+=== Configuration Sections
+
+==== Top-Level Properties
+
+[#debezium-operator-schema-reference-debeziumserverspec]
+Used in: <<debezium-operator-schema-reference-debeziumserver, `+DebeziumServer+`>>
+
+
+.DebeziumServerSpec properties
+[cols="20%a,25%s,15%a,40%a",options="header"]
+|===
+| Property | Type | Default | Description
+| [[debezium-operator-schema-reference-debeziumserverspec-image]]<<debezium-operator-schema-reference-debeziumserverspec-image, `+image+`>> | String |  | Image used for Debezium Server container. This property takes precedence over version.
```

Comment:
```suggestion
| [[debezium-operator-schema-reference-debeziumserverspec-image]]<<debezium-operator-schema-reference-debeziumserverspec-image, `+image+`>> 
| String 
|No default value 
| Image used for Debezium Server container. This property takes precedence over the version property.
```
Although I verified that the tables as currently defined in asciidoc do render correctly, to facilitate possible future editing, I'd recommend that you place the markup for each table cell on a separate line, as in the preceding suggestion. 
Also, where a property has no default value, add text that makes that explicit.